jobs/ci-kubernetes-e2e-gce-5000-performance.env

```diff
@@ -0,0 +1,35 @@
+### provider-env
```

Comment:
Also the high level comment - you are basically renaming the "gce-enormous-cluster" job - instead of having a new job, let's rename the old one - it is so much easier to review.